src/modules/pos/validator/components/UnlockBalanceForm/unlockBalanceTable.test.js

```diff
@@ -1,38 +1,40 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import { calculateSentStakesAmount, calculateUnlockableAmount } from '@wallet/utils/account';
+import {calculateSentStakesAmount, calculateUnlockedAmount, getLockedPendingUnlocks} from '@wallet/utils/account';
 import { mockSentStakes, mockUnlocks } from '@pos/validator/__fixtures__';
 import BalanceTable from './BalanceTable';
 
-describe('unlock transaction Status', () => {
+describe('unlockBalanceTable', () => {
   let wrapper;
   
-  const pendingUnlockableUnlocks = mockUnlocks.data.pendingUnlocks;
+  const pendingUnlocks = mockUnlocks.data.pendingUnlocks;
   const stakes = mockSentStakes.data.stakes;
   const currentBlockHeight = 5000;
 
+  const lockedPendingUnlocks = getLockedPendingUnlocks(pendingUnlocks);
+
   const props = {
     sentStakesAmount: calculateSentStakesAmount(stakes),
-    unlockableAmount: calculateUnlockableAmount(pendingUnlockableUnlocks),
+    unlockedAmount: calculateUnlockedAmount(pendingUnlocks),
     currentBlockHeight,
-    pendingUnlockableUnlocks,
+    lockedPendingUnlocks,
   };
 
   it('renders properly', () => {
     wrapper = mount(<BalanceTable {...props} />);
     expect(wrapper).toContainMatchingElement('.lock-balance-amount-container');
     expect(wrapper.find('.locked-balance').text()).toEqual('280 LSK');
     expect(wrapper.find('.available-balance').text()).toEqual('4,550 LSK');
-    expect(wrapper.find('.unlocking-balance')).toHaveLength(pendingUnlockableUnlocks.length);
+    expect(wrapper.find('.unlocking-balance')).toHaveLength(lockedPendingUnlocks.length);
   });
 
   it('should not show pendingUnlockableUnlocks if undefined', () => {
```

Comment:
@oskarleonard `pendingUnlockableUnlocks` should also be renamed in the test description here